core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -73,26 +73,27 @@
       </div>
       <!-- This is an ng-if, so that the preview loads the latest version of the exploration each time the tab is accessed. -->
       <div ng-if="getActiveTabName() === 'preview'">
-        {% include 'pages/exploration_editor/preview_tab/preview_tab.html' %}
+        <preview-tab></preview-tab>
       </div>
 
       <div ng-show="getActiveTabName() === 'stats'">
-        {% include 'pages/exploration_editor/statistics_tab/statistics_tab.html' %}
+        <statistics-tab></statistics-tab>
       </div>
 
       <improvements-tab ng-if="getActiveTabName() === 'improvements'">
```

Comment:
Please match this with the surrounding style (`<div ng-if><tab></tab></div>`) of match the surrounding style with this one.

Reply:
Done!